extension/chrome/scripts/content.js

```diff
@@ -148,7 +148,7 @@ var doLogin = function(data, sendResponse) {
 
 
 var authSuccess = function(data) {
-    var api_key_input = document.getElementsByTagName('input')[0];
+    var api_key_input = document.querySelector('input[name="value"]');
```

Comment:
Can you please upgrade the version and update the version information in `README.md` file? So that 6.0.16 can use the current version and 6.0.17+ will use your changes.

You can take a reference from https://github.com/delphix/Delphix-Masking-API-Client-Extension/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5